libvast/src/transform.cpp

```diff
@@ -173,7 +173,6 @@ caf::expected<std::vector<table_slice>> transformation_engine::finish() {
   // is finished: If a copy was made, `batch` will hold the only reference
   // to its underlying table slice, but the RecordBatches created by the
   // transform step will most likely reference some of same underlying data.
```

Comment:
The comment isn't needed anymore either.

Reply:
Do you still want to remove it in this PR?

Reply:
Yes I should—totally forgot that I left this comment here as a reminder to self. Will fix in a minute.